src/test/java/com/iexec/standalone/configuration/PublicConfigurationServiceTests.java

```diff
@@ -24,8 +24,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
This wildcard should not have happened for 2 static imports.
How is your IDE configured ?